src/utils.rs

```diff
@@ -290,7 +342,7 @@ impl Style {
 
     /// Adds a attr.
     #[inline]
-    pub fn attr(mut self, attr: Attribute) -> Self {
+    pub const fn attr(mut self, attr: Attribute) -> Self {
```

Comment:
Nit: please extract all the `const` qualifiers into a separate commit, since they're unrelated.